ngraph/frontend/onnx_import/src/core/transform.cpp

```diff
@@ -0,0 +1,123 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <onnx/defs/function.h>
+#include <onnx/defs/schema.h>
+
+#include "model.hpp"
+#include "transform.hpp"
+
+#include "ngraph/file_util.hpp"
+#include "onnx_import/ops_bridge.hpp"
+
+void ngraph::onnx_import::transform::expand_onnx_functions(ONNX_NAMESPACE::ModelProto& model_proto)
+{
+    auto graph_proto = model_proto.mutable_graph();
+
+    for (int i = 0; i < graph_proto->node().size(); ++i)
+    {
+        auto node = graph_proto->node().Get(i);
```

Comment:
You can use auto& here

Reply:
I actually can't use a const ref here, because of `FunctionBodyBuildContextImpl` used below.